browser/css/jsdialogs.css

```diff
@@ -1906,3 +1906,9 @@ kbd,
 .jsdialog-container [id^='info-modal-label'] {
 	display: flex;
 }
+
+input.ui-timefield {
+	width: 7.3em;
+	height: 1.7em;
```

Comment:
is the height unified with other widgets: edit/input field, spinfields ?

Reply:
No, I should probably use a min-height here.
I picked this height just so that it looks ok:
![image](https://github.com/CollaboraOnline/online/assets/620941/06320f73-3676-4355-bcb6-bee05eb19ebb)

Reply:
ok, maybe left it as it is now. We will create issue and unify fields height at some point I think.

Reply:
With unified:

![image](https://github.com/CollaboraOnline/online/assets/620941/15291d44-e01a-4909-bde4-d3ec156c9f16)

And then the border tuned:

![image](https://github.com/CollaboraOnline/online/assets/620941/cbfe322b-db5d-47c4-a5c3-2170316060c8)

Reply:
I went with the last screenshot styling.